classworks/Tatyana/01_git/result.txt

```diff
@@ -0,0 +1 @@
+https://github.com/PogudoTanya
```

Comment:
Please provide a link to the **repository** that you've made for solid-train exercise

Reply:
@PogudoTanya